README.md

```
@@ -1,12 +1,22 @@
 # 2025-DEVSONG-Server
+<img width="1280" height="720" alt="devsong_thumbnail" src="https://github.com/user-attachments/assets/69dd039f-0410-49f4-91fb-c2d0cf443fff" />
 
-## 🏄🏻‍♀️ 개발 팀 소개
+## 🏄🏻‍♀️ Team
 
+| name                                        | major        | role | Email                |
+| -------------------------------------------- | -------------- | ------ | -------------------- |
+| [CHAEAH WOO](https://github.com/chaeeah) | Computer Science | PM / BE | chaeah23@sookmyung.ac.kr |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)